### PR TITLE
Add AWS CloudWatch integration

### PR DESCRIPTION
**Pull request by laysauchoa**
Add AWS CloudWatch integration: logs and metrics

AWS CloudWatch logs: https://help.aiven.io/en/articles/4134821-sending-service-logs-to-aws-cloudwatch
AWS CloudWatch metrics: https://help.aiven.io/en/articles/5144867-aiven-service-metrics

Fixes: #963 
Fixes: #964 
Fixes: https://github.com/aiven/devportal/issues/1064
The original articles are linked in two places in the WebAiven Console.

Service Integrations -> CloudWatch logs
Service Integrations -> CloudWatch metrics

### CODE REVIEW

File: docs/integrations/cloudwatch/cloudwatch-logs.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Send logs to Amazon CloudWatch
```
Comment (laysauchoa) [inline docs/integrations/cloudwatch/cloudwatch-logs.rst:1]: cc: @TibsAtWork  is there a way to get rid off this sentenced case when using proper names?

File: docs/integrations/cloudwatch/cloudwatch-logs.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+3. Create the endpoint between your Aiven account and AWS CloudWatch.
+
+.. code:: bash
+
+   avn service integration-endpoint-create --project PROJECT \
+      -d ENDPOINT_NAME -t external_aws_cloudwatch_logs \
+      -c log_group_name=LOG_GROUP_NAME \
+      -c access_key=AWS_ACCESS_KEY\
+      -c secret_key=AWS_SECRET_ACCESS_KEY \
+      -c region=AWS_REGION
```
Comment (laysauchoa) [inline docs/integrations/cloudwatch/cloudwatch-logs.rst:113]: I was not able to find in the `avn service types -v` those customizations options from the Aiven CLI itself. They are mentioned but not listed anywhere. Therefore, I am thinking to keep this section with a step by step how to do this from the Aiven CLI.

**Comment by laysauchoa** [comment]
I'm willing to keep those two articles in one place for the logs integration. But if you think there is a reason to split in two articles, I'll consider. I have wrote all the steps to perform this task from the Aiven CLI. This information is missing in the Aiven CLI itself or I could not find it. Maybe something we can add in the future about it.

**Comment by laysauchoa** [comment]
I've rebased with main branch due to the latest commits that have been merged.

**Comment by laysauchoa** [comment]
ping @kevinbowman-aiven

File: docs/integrations/cloudwatch/index.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+You can send the metrics from any or all of your Aiven services to CloudWatch. You can specify the namespace where your metrics should be send or one will be created for you. 
+
+Find out :doc:`how to send your Aiven service metrics to AWS CloudWatch </docs/integrations/cloudwatch/cloudwatch-metrics>`.
+
+CloudWatch logs
+---------------
+
+You can send any of your Aiven service logs to your Amazon CloudWatch logs service. 
+
+Check out our tutorial on :doc:`how to send your Aiven service logs to AWS CloudWatch </docs/integrations/cloudwatch/cloudwatch-logs>`.
```
Comment (kevinbowman-aiven) [inline docs/integrations/cloudwatch/index.rst:18]: This doesn't seem to have linked properly in the preview at https://deploy-preview-961--developer-aiven-io-preview.netlify.app/docs/integrations/cloudwatch/index.html

File: docs/integrations/cloudwatch/list-cloudwatch-logs.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Send logs to Amazon CloudWatch
+==============================
+
+Browse through information related to sending your Aiven service logs to your `Amazon (AWS) CloudWatch <https://aws.amazon.com/cloudwatch/>`_.
```
Comment (kevinbowman-aiven) [inline docs/integrations/cloudwatch/list-cloudwatch-logs.rst:4]: Should there be some files for this to link to?  It looks like an empty section at the moment.

File: docs/integrations/cloudwatch/index.rst (hunk 3 above)
Comment (laysauchoa) [reply docs/integrations/cloudwatch/index.rst]: Oh you are right. I forgot to commit some files and ended up loosing it. fixed now!

File: docs/integrations/cloudwatch/cloudwatch-metrics.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+2. Configure the settings for the new endpoint:
+   
+   * Your AWS account **Region**.
+  
+   * Your AWS credentials: **Access Key** and **Secret Key**.
+
+   * **Endpoint name** is how you will refer to the AWS CloudWatch metrics integration when linking it to an Aiven service.
+  
+   * **CloudWatch Namespace** where your metrics can be organized in different spaces. This field is `Optional` and if not provided, one will be created for you.
```
[the remark this answers is not in the document]
Comment (laysauchoa) [reply docs/integrations/cloudwatch/cloudwatch-metrics.rst]: Solved, thanks for observing this

**Comment by laysauchoa** [comment]
Ready to re-review

**Comment by laysauchoa** [comment]
ping @kevinbowman-aiven would be possible to check once more? thank you!

**Comment by kevinbowman-aiven** [comment]
Hi @laysauchoa, sure thing!  It'll probably be Friday before I get a chance, but I'll try to look this week.

**Comment by laysauchoa** [comment]
ping @kevinbowman-aiven 🙏

File: docs/integrations/cloudwatch/cloudwatch-logs-cli.rst [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+  * - Variable
+    - Description
+  * - ``PROJECT``
+    - The Aiven project where your endpoint is saved.
+  * - ``ENDPOINT_ID``
+    - Reference name for this log integration when linking it to other Aiven services.
+  * - ``AIVEN_SERVICE_NAME``
+    - The Aiven service name that you want to send the logs from.
+
+1. Send logs from the Aiven service to AWS CloudWatch by running:
```
Comment (kevinbowman-aiven) [inline docs/integrations/cloudwatch/cloudwatch-logs-cli.rst:107]: Should the number "1" be a number "6" here?  It's funny that it jumps from 5 to 1

**Comment by kevinbowman-aiven** [comment]
From a read through it looks good (I've added 1 minor comment but that's all).  I haven't executed all of the steps myself yet, I'll try to do that later today, but if you want to get this published and you're confident about the steps (and from my understanding they look correct) then I'm happy with that (and if I find any problems when I get a chance to test it fully then I'll submit an issue/PR).

**Comment by laysauchoa** [comment]
Hi @kevinbowman-aiven I've tested the steps myself, so I feel confident about their being correct. If you find something that could improve, it would be nice and you can add in a separate PR for it as you suggested. Thank you very much!

**Review by kevinbowman-aiven — APPROVED** [review]

**Pull request merged**